base/compiler/abstractinterpretation.jl

```diff
@@ -147,22 +165,23 @@ function abstract_call_gf_by_type(interp::AbstractInterpreter, @nospecialize(f),
             if edge !== nothing
                 push!(edges, edge)
             end
+            # try constant propagation with argtypes for this match
+            # this is in preparation for inlining, or improving the return result
+            this_argtypes = applicable_argtypes === nothing ? argtypes : applicable_argtypes[i]
+            const_this_rt, const_result = abstract_call_method_with_const_args(interp, this_rt, f, this_argtypes, match, sv, edgecycle)
```

Comment:
`edgecycle` hasn't been properly computed yet, so it cannot be used. But I think you meant `edgecycle1` here, since the other arguments are all call-specific?

Reply:
Ah, good catch. I overlooked this. Thanks.